packages/ingest-mongodb-public/src/sources/index.ts

```diff
@@ -53,9 +52,9 @@ const mongoDbUniversitySourceConstructor = async () => {
     sourceName: "mongodb-university",
     baseUrl: "https://api.learn.mongodb.com/rest/catalog",
     apiKey: universityDataApiKey,
-    tiCatalogFilterFunc: filterOnlyPublicActiveTiCatalogItems,
-    metadata: {
-      tags: ["transcript"],
+    tiCatalogItemsConfig: {
```

Comment:
```suggestion
    tiCatalogItems: {
```

nit: prefer not to have word `config` if not needed